lib/mayaUsd/resources/scripts/mayaUsdLibRegisterStrings.py

```diff
@@ -41,7 +41,7 @@ def mayaUsdLibRegisterStrings():
     # mayaUsdCacheMayaReference.py
     register('kButtonNewChildPrim', 'New Child Prim')
     register('kButtonNewChildPrimToolTip', 'If selected, your Maya reference will be defined in a new child prim. This will enable\nyou to work with your Maya reference and its USD cache side-by-side.')
-    register('kCacheFileWillAppear', 'Cache file will appear\non parent prim:')
+    register('kCacheFileWillAppear', 'Cache file will\nappear on parent\nprim:')
```

Comment:
Suggestion from design - use 3 lines for this label (instead of 2).

Reply:
Clever use of amazing new line technology.